ios/RNDocumentPicker/RNDocumentPicker.m

```diff
@@ -79,18 +87,21 @@ - (dispatch_queue_t)methodQueue
 
 - (NSMutableDictionary *)getMetadataForUrl:(NSURL *)url error:(NSError **)error
 {
-    __block NSMutableDictionary* result = [NSMutableDictionary dictionary];
+    __block NSMutableDictionary *result = [NSMutableDictionary dictionary];
     
+    if (mode == UIDocumentPickerModeOpen)
+        [urls addObject:url];
     [url startAccessingSecurityScopedResource];
     
     NSFileCoordinator *coordinator = [[NSFileCoordinator alloc] init];
     NSError *fileError;
     
     [coordinator coordinateReadingItemAtURL:url options:NSFileCoordinatorReadingResolvesSymbolicLink error:&fileError byAccessor:^(NSURL *newURL) {
+        
         if (!fileError) {
-            [result setValue:newURL.absoluteString forKey:FIELD_URI];
+            [result setValue:((mode == UIDocumentPickerModeOpen) ? url : newURL).absoluteString forKey:FIELD_URI];
```

Comment:
just wondering, why is this needed? thanks!

Reply:
Because sometimes those two urls where a little bit different during my tests and I wasn't able to save documents with the new one.